### PR TITLE
removeExtraQuotationMarks

### PR DESCRIPTION
**Pull request by nvdaes**
### Summary of the issue

Add-ons cannot be submitted to the store, since an extra quotation mark is added in metadata values. The cause of this is not determined, but maybe due to the [transition to Node 20 for GitHub Actions](https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20).

### Development strategy

- GitHub actions have been updated to their last versions.
- - The action used to merge has been removed, in favour of a GH CLI command.
- In step Create JSON submission from issue, just two quotation marks havebeen used to create the json file, removing the extra quotation mark from arguments of the corresponding command.
- 
### Testing performed

https://github.com/nvdaes/addon-datastore/actions/runs/7766928875

### Affected submitters

@aaclause  @josephsl @ABuffEr

Please, when this PR is merged, resubmit your add-ons.

### CODE REVIEW

**Comment by nvdaes** [comment]
### Summary of the issue

Add-ons cannot be submitted to the store, since an extra quotation mark is added in metadata value. The cause of this is not determined, but maybe due to the transition to Node 20 made by GitHub for GitHub Actions.

### Development strategy

- 
- GitHub Script action has been updated to v7, to avoid annotations due to the transition to Node 20.
- In step Create JSON submission from issue, just two quotation marks havebeen used to create the json file, removing the extra quotation mark from arguments of the corresponding command.
### Testing performed

https://github.com/nvdaes/addon-datastore/issues/908


https://github.com/nvdaes/addon-datastore/actions/runs/7761491660

### Affected submitters


@aaclause  @josephsl @ABuffEr

**Comment by nvdaes** [comment]
I'm update all used actions to the latest versions to avoid warnings. I'll use noBeepsSpeechMode to provide links here confirming that this works as expected

**Comment by nvdaes** [comment]
Additionally, I've updated all actions to their latest version. For the merge action, I used gh CLI since the merge action shows warning and seems not to be maintained.

https://github.com/nvdaes/addon-datastore/actions/runs/7766928875

**Comment by nvdaes** [comment]
Note that I haven't tested all possible scenarios, i.e., cases where failures are produced due to wrong submissions, where the [Create or update comment](https://github.com/peter-evans/create-or-update-comment) action is used. I've updated it from v2 to v4.
Also, the upload artifact action is also updated to v4.
Moreover, in [close-pull](https://github.com/peter-evans/close-pull) and [close-issue](https://github.com/peter-evans/close-issue) documentation, the author of these actions (Peter Evans), says that they are no longer necessary (though they can be used if desired), since the same functionality can be achieved with GH CLI.

File: addons/noBeepsSpeechMode/5.5.0.json [hunk 1]
```diff
@@ -0,0 +1,192 @@
+{
```
Comment (seanbudd) [inline addons/noBeepsSpeechMode/5.5.0.json:1]: please avoid submitting new add-ons in this PR

File: discussions.json [hunk 2]
```diff
@@ -290,5 +290,9 @@
   "openLinkWith": {
     "discussionId": "D_kwDODeB9Us4AXcT_",
     "discussionUrl": "https://github.com/nvaccess/addon-datastore/discussions/2438"
+  },
```
Comment (seanbudd) [inline discussions.json:293]: please remove this too

**Comment by nvdaes** [comment]
I've addressed your review, @seanbudd . In fact, I submitted the add-on and the metadataforits discussion unintentionally due to testing. Hope this is right now.

**Comment by seanbudd** [comment]
can you please update the PR description to be more descriptive of the changes

**Review by seanbudd — APPROVED** [review]

**Comment by nvdaes** [comment]
> can you please update the PR description to be more descriptive of the changes

Done

**Comment by nvdaes** [comment]
Thanks @seanbudd , I'll announce this on the add-ons mailing list so that people continue submitting add-ons

**Pull request merged**